universal-login-wallet/cypress/integration/createAccount.test.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path="../support/index.d.ts" />
```

Comment:
remove comment

Reply:
This is not a comment, it won't work without it